_posts/2021-08-03-cockpit-250.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Cockpit 250
+author: garrett
+date: '2021-08-03'
+tags: cockpit, machines
+slug: cockpit-250
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 250 and cockpit-machines 249:
+
+
+## shell: Change user icon in the top menu
+
+It is about 'Session' and not about user.
+
+Fixes #16173
+
+![Screenshot from 2021-08-02 12-52-18s1]({{ site.baseurl }}/images/250-shell-change-user-icon-in-the-top-menu.png)
+
+![screenshot of change user icon in the top menu]({{ site.baseurl }}/images/250-shell-change-user-icon-in-the-top-menu-2.png)
+
+## Shell: Improve admin switcher and session menu
+
+Cockpit has a "limited access" mode with lowered privileges, where browsing generally works, but changing things that require administration rights generally does not. 
+
+If someone is stuck in "limited access" mode without knowing how to switch, it would appear that Cockpit is "broken" or cannot perform tasks it should be able to do. To make it more obvious how to switch modes, Cockpit now has several "eye-catching" changes to make switching easier to notice. These changes are in the shell, which is visible on every page within Cockpit, and the overview page's alert.
+
+- On the overview alert, a warning color has been used to draw attention
+- A lock icon has been used in both the overview alert and the shell privilege escalation icon
+- The shell's escalation action now resembles a button
+
+![screenshot of improve admin switcher and session menu]({{ site.baseurl }}/images/250-shell-improve-admin-switcher-and-session-menu.png)
+
+
+All of the above changes only affect "limited access" mode. Normal administration mode looks the same as before.
+
+The menu on the top right is now called "Session" instead of showing an user icon. This clarifies that the settings in that menu apply to remote hosts as well, which possibly run as a different user.
+
+*Thanks to Verhoeckx for suggesting these design changes!*
+
+## Machines: Migration of a virtual machine
+
+The Machines page can now migrate a VM to a different machine. It supports live, offline, and temporary migration. Commonly, the VM's storage is on shared storage, which avoids having to copy the data to the remote machine and makes the migration process much faster and more robust. As it is not possible to automatically detect whether the storage is shared, the user has to explicitly tell whether the storage must be copied.
```

Comment:
```suggestion
The Machines page can now migrate a VM to a different machine. It supports live, and temporary migration. Commonly, the VM's storage is on shared storage, which avoids having to copy the data to the remote machine and makes the migration process much faster and more robust. As it is not possible to automatically detect whether the storage is shared, the user has to explicitly tell whether the storage must be copied.
```
The offline migration is not supported.

Reply:
Thanks @skobyda -- I updated the text in https://github.com/cockpit-project/cockpit-machines/pull/15 as that is the primary truth, and I'll update this PR once again (the second PR with the "Session" menu slipped in)